Sources/MongoSwift/BSON/BsonEncoder.swift

```diff
@@ -1,348 +1,557 @@
 import Foundation
```

Comment:
FYI I would just ignore this file's entire diff... all I did was rename `BsonEncoderNew.swift` -> `BsonEncoder.swift`

Reply:
actually you should look at `encodeIfPresent`, that was a new addition